biothings/utils/common.py

```diff
@@ -649,6 +649,17 @@ def find_classes_subclassing(mods, baseclass):
                 classes.append(klass)
     return classes
 
+
+def sanitize_tarfile(tar_object, directory):
```

Comment:
@remoteeng00 Please leave a doc string to this function to explain why and how it works